news/_posts/core-weekly/2019-03-13-coreweekly-week-10-2019.md

```diff
@@ -0,0 +1,117 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 10 of 2019"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2019-03-13 14:30:00
+authors: [ AntoineThomas ]
+icon: icon-calendar
+tags:
+ - core-weekly
+---
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 04th to Sunday 10th of March 2019.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+
+## General messages
+
+Dear Developers,
+
+An important step for the PrestaShop open source project has been completed last week and it is available today: the nighlty builds and tests dashboard. This is a new website: https://nightly.prestashop.com/.
+
+Each night, a new build is created with the newly merged code. Then, the automatic tests are running for this build, and a report is created. It will help contributors to improve the quality of PrestaShop.
```

Comment:
We could say that the tests run are different from the suite on Travis. These are additional tests covering more features of PrestaShop which may require several hours to complete. That's why we don't execute them on each pull-request for instance.

Reply:
Good idea

Reply:
what do you think of the new version ?